example/lib/Pages/AlphaPickerPage.dart

```diff
@@ -2,48 +2,42 @@ import "package:flutter/material.dart";
 import 'package:flutter_hsvcolor_picker/flutter_hsvcolor_picker.dart';
 
 class AlphaPickerPage extends StatefulWidget {
-
   @override
-  AlphaPickerPageState createState() => new AlphaPickerPageState();
+  AlphaPickerPageState createState() => AlphaPickerPageState();
 }
 
 class AlphaPickerPageState extends State<AlphaPickerPage> {
-    
-  int value =0;
-  void onChanged(int value) => this.value=value;
-  
+  int value = 0;
+  void onChanged(int value) => this.value = value;
+
   @override
   Widget build(BuildContext context) {
-    return new Center(
-      child: new Container(
-        width: 260,
-        child: new Card(
-          shape: new RoundedRectangleBorder(borderRadius: BorderRadius.all(Radius.circular(0.0))),
-          elevation: 2.0,
-          child: new Padding(
-            padding: const EdgeInsets.all(10),
-            child: new Column(
-              mainAxisSize: MainAxisSize.min,
-              children: <Widget>[
-                new Text(
-                  this.value.toString(),
-                  style: Theme.of(context).textTheme.display1,
-                ),
-                new Divider(),
+    return Center(
+        child: Container(
+            width: 260,
+            child: Card(
+                shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.all(Radius.circular(0.0))),
+                elevation: 2.0,
+                child: Padding(
+                    padding: const EdgeInsets.all(10),
+                    child: Column(
+                        mainAxisSize: MainAxisSize.min,
+                        children: <Widget>[
+                          Text(
+                            this.value.toString(),
+                            style: Theme.of(context).textTheme.headline4,
+                          ),
+                          Divider(),
 
-                ///---------------------------------
-                new AlphaPicker(     
-                  alpha: this.value,
-                  onChanged: (value)=>super.setState(()=>this.onChanged(value)),
-                )
-                ///---------------------------------
+                          ///---------------------------------
+                          AlphaPicker(
+                            alpha: this.value,
+                            onChanged: (value) =>
+                                super.setState(() => this.onChanged(value)),
+                          )
 
-              ]
-            )
-          )
-        )
-      )
-    );
+                          ///---------------------------------
+                        ])))));
```

Comment:
Add trailing commas here and format to avoid `)))`, same as other places like this.